Backpack/build.gradle

```diff
@@ -80,6 +80,7 @@ dependencies {
   testImplementation rootProject.ext.threeTenAbp
 
   lintPublish project(':backpack-lint')
+  implementation project(':Backpack-Common')
```

Comment:
a nit on the module names - module names are generally lowercase - `Backpack` is the odd one out (maybe we can fix that too)